### ISSUE TITLE
Extra semicolon causes false positive in `block-closing-brace-newline-before`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1212]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Strange behavior

> Which rule, if any, is this issue related to?

e.g. `block-closing-brace-newline-before`

> What CSS is needed to reproduce this issue?

e.g.

```css
.no-padding {
    padding: 0;;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "block-closing-brace-newline-before": "always",
    "no-extra-semicolons": true,
  }
}
```

> Which version of stylelint are you using?

e.g. `6.3.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

One error

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
_utilities.scss
2:14	⚠  Unexpected extra semicolon (no-extra-semicolons) [stylelint]
6:17	⚠  Expected newline before "}" (block-closing-brace-newline-before) [stylelint]

```

**Comment by jeddy3** [comment]
I'm assuming both errors go away when you remove the extra colon?

If so, it's a low priority bug.

**Comment by evilebottnawi** [comment]
Close. Merged. Ref: https://github.com/stylelint/stylelint/pull/1217
